src/dhtproto/client/legacy/internal/helper/RetryHandshake.d

```diff
@@ -113,32 +86,34 @@ class RetryHandshake
 
         this.timer = new TimerEvent(&this.tryHandshake);
 
+        this.log = Log.lookup(format("RetryHandshake instance 0x{:x}",
```

Comment:
Do you think there's any point to formatting the logger name like this? Theoretically, a user might have >1 DHT client, each with their own `RetryHandshake` instance.

Reply:
I don't mind one way or the other, but it might be nicer to allow the user to specify a custom identifier if that is the intended use-case.

Reply:
I suggest having structured logger name:

```
RetryHandshake.Instance 0x{:x}
```

Reply:
Why use a `.` between the first two words but not the second?

Reply:
I think `RetryHandshake.0x{:x}` also works. The trick is that the config file could then do `LOG.RetryHandshake.*`

Reply:
(iow, I'm not sure if the globbing works on the arbitrary point in the logger name)

Reply:
Oh is `.` a magic character for the logger framework? (I wasn't sure if it is or not.)

Reply:
Updated.